kubernetes-1.24.yaml

```diff
@@ -167,7 +167,27 @@ subpackages:
         runs: |
           mkdir -p ${{targets.subpkgdir}}/usr/bin/
           CFLAGS="$CFLAGS -static -DVERSION=${{vars.pause-version}}-${{package.version}}"
-          gcc -o "${{targets.subpkgdir}}"/usr/bin/pause linux/pause.c
+          gcc -o "${{targets.subpkgdir}}"/usr/bin/pause-3.7 linux/pause.c
+
+  - name: kubernetes-1.24-default
+    description: "Compatibility package to set 1.24 as the default kubernetes, and add packages to their shortened path"
+    dependencies:
+      runtime:
+        - kubectl=1.24
+        - kubelet=1.24
+        - kube-scheduler=1.24
+        - kube-proxy=1.24
+        - kube-controrller-manager=1.24
+        - kube-apiserver=1.24
```

Comment:
```suggestion
```

I'm second guessing myself here, I think these should be empty, so that things like:


```yaml
environment:
  contents:
    packages:
      - kubectl-1.24
      - kubernetes-1.24-default
```

can give the desired effect of defaulting `kubectl`, without pulling in all the other components?

Reply:
Yeah I think I agree with that.